drivers/sensor/tmd26xx/Kconfig

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2023 PHYTEC Messtechnik GmbH
+# Copyright(c) 2023 Carl Zeiss Meditec AG
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig TMD26XX
+	bool "OSRAM-AMS TMD26XX Proxmimity Sensor"
+	default y
+	depends on DT_HAS_AMS_TMD2635_ENABLED || DT_HAS_AMS_TMD2620_ENABLED
+	select I2C
+	help
+	  Enable driver for TMD26XX Sensor (e.g. TMD2635, TMD2620).
+
+if TMD26XX
+
+choice
+	prompt "Trigger Mode"
+	default TMD26XX_TRIGGER_NONE
+	help
+	  Specify the type of triggering to be used by the driver.
+
+config TMD26XX_TRIGGER_NONE
+	bool "No Trigger"
+
+config TMD26XX_TRIGGER_GLOBAL_THREAD
+	bool "Use global thread"
+	depends on GPIO
+	select TMD26XX_TRIGGER
+
+endchoice # Trigger Mode
+
+config TMD26XX_TRIGGER
+	bool
+
+choice	TMD26XX_SELECT_MODE
+	prompt "Select Sensor type"
+	default TMD26XX_SELECT_NONE
+	help
+	  Specify the type of triggering to be used by the driver.
+
+config TMD26XX_SELECT_NONE
+	bool "select none default support tmd2620"
+
+config TMD26XX_SELECT_TMD2635
+	bool "supports tmd2635"
+	depends on TMD26XX
+	select TMD2635
+
+endchoice
+
+config TMD2635
+	bool
```

Comment:
is this kconfig variable required? you have a choice between the two

Reply:
this variable is required otherwise the driver will not be initialized